data/json/items/gun/50.json

```diff
@@ -67,7 +67,9 @@
     },
     "material": [ "steel", "wood" ],
     "clip_size": 1,
+    "modes": [],
```

Comment:
@mugling current master branch, deleted clipsize line from m2browning_sawn, picked new game, random character, got "warnings for type m2browning_sawn: missing or duplicate clip_size or magazine, press any key", pressing a key turns the screen black, cpu spikes and "program is not responding, close or wait" windows dialog pops up
loading existing character or choosing play now works as expected, after the warning message the game starts up fine
nothing in debuglog in either case